src/Common/AbstractProcessor.php

```diff
@@ -40,10 +40,7 @@
      * @param EventDispatcherInterface $dispatcher
      * @param LoggerInterface          $logger
      */
-    public function __construct(
-        EventDispatcherInterface $dispatcher,
-        LoggerInterface $logger
-    ) {
+    public function __construct(EventDispatcherInterface $dispatcher, LoggerInterface $logger) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this open curly brace to the beginning of the next line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1808)